### DISCUSSION TITLE
canvas login

### DISCUSSION THREAD

**Discussion by llAstaroth** [thatskymod/Sky-CotL-Scripts#37] [General]
Good good taskymod I hope and you answer me.

I wanted to ask if the canvas login problem is already fixed?

**Comment by Kekechann** [comment]
Not yet!!

**Comment by thatskymod** [comment]
The notorious _'Session error: no authentication info'_ encountered during Google login has been addressed in Canvas version 1.3.9, released on August 1st, 2023. This long-standing problem is now considered resolved.

The latest and greatest releases of Canvas are available here: https://github.com/RomanChamelo/Canvas-Open-Source/releases

Thanks for your patience while this was being worked on.

**Comment by aph3Iios** [comment]
hello ! wondering on why sky beta on canvas does not support google login?

**Reply by thatskymod** [reply]
Because Google login has proven problematic, it has been removed from Canvas for Sky Beta. So you are right in noting that it is missing from the list of available login methods.
